frontend/views/components/Avatar.vue

```diff
@@ -39,9 +39,11 @@ export default {
       this.$refs.img.onload = function () { URL.revokeObjectURL(this.src) }
       this.objectURL = URL.createObjectURL(blob)
     },
-    async setBlobURL (newBlobURL) {
-      const oldBlobURL = this.blobURL
-      if (!newBlobURL) return // don't do anything if newBlobURL is blank
+    async setBlobURL (newBlobURL, oldBlobURL) {
```

Comment:
I'm not sure why we pass 'oldBlobURL' variable into the function if this is a variable already available in the component as this.blobURL (assuming that blobURL was working before)